scripts/check_deployment_secrets.py

```diff
@@ -15,17 +13,15 @@
 
 #1
 For the json returned from the secret in GOOGLE_APPLICATION_SECRETS_SECRETS_NAME:
-    `auth_uri` should be in ['https://auth.data.humancellatlas.org/authorize',
-                             'https://auth.dev.data.humancellatlas.org/authorize']
-    `token_uri` should be in ['https://auth.data.humancellatlas.org/oauth/token',
-                              'https://auth.dev.data.humancellatlas.org/oauth/token']
+    `auth_uri` should be 'https://accounts.google.com/o/oauth2/auth'
+    `token_uri` should be 'https://oauth2.googleapis.com/token'
 
 #2
 For the json returned from the secret in GOOGLE_APPLICATION_CREDENTIALS_SECRETS_NAME:
-    `project_id` should be `human-cell-atlas-travis-test`
+    `project_id` should be `platform-hca`
     `type` should be `service_account`
     `client_email` should be the user account returned from the terraform output "service_account".
-                   For example: dev should be `travis-test@human-cell-atlas-travis-test.iam.gserviceaccount.com`.
+                   For example: dev should be `travis-test@platform-sc.iam.gserviceaccount.com`.
```

Comment:
This is not actually correct... currently the GCP project ID is still `platform-hca`, not `platform-sc`, and project ID is what's used to create service account emails.

```suggestion
                   For example: dev should be `travis-test@platform-hca.iam.gserviceaccount.com`.
```